### ISSUE TITLE
Travis CI: Builds have been temporarily disabled for private and public repositories due to a negative credit balance.

### ISSUE THREAD

**Issue by mathias-luedtke** [ros-industrial/ros_industrial_issues#71]
I now get a banner

> Builds have been temporarily disabled for public repositories due to a negative credit balance. Please go to the Plan page to replenish your credit balance or alter your Consume paid credits for OSS setting.

I have now unchecked this option:

![image](https://user-images.githubusercontent.com/727227/102009345-2947d080-3d37-11eb-8cce-1ea62ac1b516.png)

**Comment by mathias-luedtke** [comment]
Ah, Travis changed its billing policy: https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing
And we only get 10K credits = 1000 min for all non-OSS-approved ROS-Industrial repositories.

**Comment by gavanderhoorn** [comment]
Yes, this is a major problem for `ros-industrial` and `ros-industrial-consortium`.

> And we only get 10K credits = 1000 min for all non-OSS-approved ROS-Industrial repositories.

I've contacted Travis to ask for an "allotment of OSS credits", but the response was that they've stopped that for now (meaning: no OSS credits for us, or anyone afaict until they resume that).

We've already migrated some repositories to GH Actions. With `industrial_ci` that's essentially a relatively easy switch.

---

Edit: I fully understand Travis CI needs to increase their revenue stream. I'm also not against paying for a good service.

However, for the *community* supported repositories the costs would be significant and not something we want to spend resources on right now.
